A stub for the graphics team goals (#16651)
PostHog/posthog.com: contents/teams/graphics/objectives.mdx

### Q2 2026 objectives

These are the primary goals the team is prioritizing this quarter. 

<details>
<summary>Goal 1 (Lottie)</summary>

- **Rationale:** Helping PostHog transition into a 2030 brand language.
- **Things we could do:** Redesigning graphics and other design assets to feel more in line with the future feeling website, and make us look and feel more like a AI native 2030 brand. 
- **We'll know we're successful when:** We have a strong and consistent brand language across the company that feels unified.

</details>

<details>
<summary>Goal 1 (Heidi)</summary>

- **Rationale:** Explore what PostHog illustrations look like in a more tangible space.
- **Things we could do:** People are usually interacting with PostHog art through a screen, let’s give them something to physically interact with, think murals and art objects.

> - **We'll know we're successful when:** Getting new eyeballs on PostHog and people talking about us online. 

Reviewer: ```suggestion
- **We'll know we're successful when:** People post pictures of the new stuff we've created online. 
```
